.github/workflows/build.yaml

```diff
@@ -50,23 +50,27 @@ jobs:
     needs: Build_and_run_cargo
     name: Run mocha tests
     runs-on: ubuntu-latest
-    container:
-      image: docker.io/mayadata/ms-buildenv:latest
-      options: --privileged -v /dev:/dev -v /bin:/host/bin -v /lib/modules:/lib/modules --cpus 2
     steps:
       - uses: actions/checkout@v2
       - uses: actions/download-artifact@v2
         with:
           name: mayastor
           path: ./target/debug
-      - run: ( cd target/debug/ && chmod a+x mayastor mayastor-client mayastor-agent spdk initiator mctl )
-      - run: ln -s /host/bin/kmod /bin/modprobe
-      - run: /bin/modprobe nbd
-      - run: /bin/modprobe xfs
-      - run: echo 8192 | tee /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
-      - run: ( cd mayastor-test && npm install)
-      - run: ( cd mayastor-test && ./node_modules/mocha/bin/mocha test_cli.js )
-      - run: ( cd mayastor-test && ./node_modules/mocha/bin/mocha test_replica.js )
-      - run: ( cd mayastor-test && ./node_modules/mocha/bin/mocha test_csi.js )
-      - run: ( cd mayastor-test && ./node_modules/mocha/bin/mocha test_nexus.js )
-      - run: ( cd mayastor-test && ./node_modules/mocha/bin/mocha test_rebuild.js )
+      - run: ( sudo modprobe nbd )
+      - run: ( sudo modprobe xfs )
+      - run: ( lsmod | grep -e xfs -e nbd )
+      - run: ( echo 2048 | sudo tee /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages )
```

Comment:
I imagine this change is because we're now running on the cloud? If so a comment would be good because if we switch back to the self hosted we'd be overriding the 8192 from the other tests (which can run in parallel)

Reply:
Done. https://github.com/openebs/Mayastor/pull/260/commits/18ee8732816be85c37efcd7d39bb8303bd0dc415